src/ci/github-actions/calculate-job-matrix.py

```diff
@@ -97,9 +97,15 @@ def find_run_type(ctx: GitHubCtx) -> Optional[WorkflowRunType]:
             "refs/heads/automation/bors/try"
         )
 
+        # Unrolled branch from a rollup for testing perf
+        # This should **not** allow custom try jobs
+        is_unrolled_perf_build = ctx.ref == "refs/heads/try-perf"
+
         if try_build:
-            jobs = get_custom_jobs(ctx)
-            return TryRunType(custom_jobs=jobs)
+            custom_jobs = []
+            if not is_unrolled_perf_build:
+                custom_jobs = get_custom_jobs(ctx)
+            return TryRunType(custom_jobs=custom_jobs)
```

Comment:
Alternatively you could remove it from the `try_build = ctx.ref in (...)` list and add a new `if ctx.ref == "refs/heads/try-perf":` block, to be consistent with the `if ctx.ref == "refs/heads/auto"` block below.

r=me with or without that since it's just a python change, unless you want somebody from infra to double check.

Reply:
An unrolled perf run is still a try build in the context of this file; while now that doesn't have any further repercussions in the code, I want to have just a single codepath that leads to `TryRunType` being created. So let's keep it like this for now and let's see if it works.